YoCode/TestCountCheck.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace YoCode
+{
+    public class TestCountCheck
+    {
+        string processName;
+        string workingDir;
+        string arguments;
+
+        public string statLine { get; set; }
+        public string Output { get; set; }
+
+        TestStats stats;
```

Comment:
This is never used by anything, is that deliberate?

Reply:
![image](https://user-images.githubusercontent.com/7968437/42572710-f5540d0a-8511-11e8-9010-08e3c654f3ad.png)

They are used to stored the results from the methods

Reply:
They are used to store those values, but those values are never read from that `stats` field. Is that deliberate?